ci/test/pipeline.template.yml

```diff
@@ -687,6 +687,17 @@ steps:
           composition: sqlancer
           args: [--runtime=600, --oracle=HAVING]
 
+  - id: data-ingest
```

Comment:
You may wish to incubate it in Nightly while the framework is under active development.